### DISCUSSION TITLE
[Android] StageWebView zoom panel

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#3061] [Q&A]
Is there a way to disable showing of this ugly magnifying panel in StageWebView after finger pinch on Android?

![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/56675407/7c215d44-966e-4be0-9630-48a0baaed6ce)

**Comment by ajwfrost** [comment]
Looks like it could be an option:
https://stackoverflow.com/a/7903262
Will have to look at how we can incorporate this setting into the ActionScript APIs...

**Reply by itlancer** [reply]
Related feature requests:
https://github.com/airsdk/Adobe-Runtime-Support/issues/2832
https://github.com/airsdk/Adobe-Runtime-Support/issues/2946
https://github.com/airsdk/Adobe-Runtime-Support/issues/2831

**Reply by MalacTheLittle** [reply]
@ajwfrost Do you have any updates on this?
Also, it is very noticeable that the smoothness of scrolling in stageWebView is slow. Can this be improved with some settings?
And, is there a list of Object parameters that can be adjusted while instantiating StageWebView?

**Reply by MalacTheLittle** [reply]
Adding `<application android:hardwareAccelerated="true" />` to app manifest fixed the issue with scrolling smoothness.

**Comment by FliplineStudios** [comment]
I'd love to be able to force it to start as being zoomed out all the way too.  For a bunch of our apps where I use a StageWebView to showcase a list of our other apps, it always starts out as zoomed in one level on Android.  Seems to be the case no matter the screen size of the device, and users have to manually click the (-) button to zoom out.

**Comment by itlancer** [comment]
Now it can be done using AIR 51.3.1.1 https://github.com/airsdk/Adobe-Runtime-Support/issues/2946#issuecomment-3969337146
